settings.gradle

```diff
@@ -4,11 +4,11 @@ pluginManagement {
   plugins {
     id "com.gradle.enterprise" version "3.6"
     id "com.gradle.common-custom-user-data-gradle-plugin" version "1.2.1"
-    id "org.asciidoctor.jvm.convert" version "3.2.0"
-    id "net.nemerosa.versioning" version "2.13.1"
+    id "org.asciidoctor.jvm.convert" version "3.3.2"
+    id "net.nemerosa.versioning" version "2.14.0"
     id "de.marcphilipp.nexus-publish" version "0.4.0"
```

Comment:
Probably, it would be good in long-term to consider switching to https://github.com/gradle-nexus/publish-plugin/, a spiritual successor of my gradle-nexus-staging-pluign and @marcphilipp's nexus-publish-plugin.

Reply:
Would you like to create a PR? For the time being, I'd like to keep the manual release step via the frontend.

Reply:
The Marc's plugin is [deprecated](https://github.com/marcphilipp/nexus-publish-plugin), so a need for migration is rather a matter of time. Nevertheless, it is not a blocker for a time being.

Reply:
Yes, I was just mentioning, that since the new plugin could automate the close and release step,t hat I'd like to do the last one (release step) myself via the frontend as a manual quality gate.